plotting.py

```diff
@@ -135,17 +154,31 @@ def plot_episodes(frequencies, episodes, mcat):
         plt.plot(xax_end, yax, 'r', linestyle=(0, (5, 10)))
 
     plt.show()
-    return
 
 
 def _legend_without_duplicate_labels(ax, fontsize=10, loc='upper right'):
+    """
+    Creates a legend without duplicate labels.
+
+    @param ax: the axis for which the legend has to be created
+    @param fontsize: the size of the font
+    @param loc: the location of the legend
+    """
     handles, labels = ax.get_legend_handles_labels()
     unique = [(h, l) for i, (h, l) in enumerate(zip(handles, labels)) if l not in labels[:i]]
     unique = sorted(unique, key=lambda x: x[1])
     ax.legend(*zip(*unique), loc=loc, fontsize=fontsize)
 
 
 def plot_alert_volume_per_episode(tid, attacker_victim, host_episodes, mcats):
+    """
+    Plots the alert volume per episode for the episodes of the given attacker-victim pair and the given mcat.
```

Comment:
Is this more or less the purpose of this method?

Reply:
No, it's more like it plots the episodes over time for each attacker victim pair. It shows a global view of the different attacker actions over time